tock/utilization/employee.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
```

Comment:
another minor thing, but to avoid confusion with the employee app, what if we called this "individual"?

Reply:
I'd like to leave as-is for now `utilization.employee` feels like a specific enough scoping to avoid confusion unless we start using this piece of work beyond utilization